File: pset1/pset1/mario.c
#include <cs50.h>
#include <stdio.h>


void mario(int h);                  //function decleration
int main(void)
{
    int height;
    do
    {
        height = get_int("Height :");
    }
    while (height < 1 || height > 8);
    mario(height);                      //function call

}
void mario(int h)                       //function to print
{
    int i;
    int k;
    int j;
    for (i = 0; i < h; i++)
    {
        for (k = h - 1 ; k > i; k--)
        {
            printf(" ");                //print empty space

        }
        for (j = 0; j <= i; j++)
        {
            printf("#");                //print '#'

        }
        
        
        printf("\n");

    }
}

File: pset6/pset6/mario.py
from cs50 import get_int


def main():
    height = check()
    for i in range(0, height):
        for k in range(height-1, i, -1):
            print(" ", end="")  # print empty space
        for j in range(0, i+1):
            print("#", end="")  # print '#'
        for z in range(2):
            print(" ", end="")
        for z in range(0, i+1):
            print("#", end="")

        print()


def check():
    height = get_int("Height : ")  # input height
    if height in range(1, 9):   # correct height
        return height
    else:
        return check()


main()